### PR TITLE
Feature/6829 countries

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[[speedup] Modal windows (My ORCID/Components) (8)](https://trello.com/c/Qej43BsL/6929-speedup-modal-windows-my-orcid-components-8)

File: src/app/cdk/pannel/panel/panel.component.ts [hunk 1]
```diff
@@ -17,12 +21,20 @@ export class PanelComponent implements OnInit {
...
   editable = true
-  constructor(private _dialog: MatDialog) {}
+  constructor(
+    private _dialog: MatDialog,
+    private _platform: PlatformInfoService
+  ) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    // if (this.editModalComponent == ModalCountryComponent) {
```
Comment (DanielPalafox) [inline src/app/cdk/pannel/panel/panel.component.ts:30]: @leomendoza123 I don't think we need this

File: src/app/cdk/modal/modal-footer/modal-footer.component.scss [hunk 2]
```diff
@@ -8,6 +8,8 @@
   display: flex;
   height: 100%;
   align-items: center;
+  padding-left: 16px;
+  padding-right: 16px;
```
Comment (DanielPalafox) [inline src/app/cdk/modal/modal-footer/modal-footer.component.scss:12]: @leomendoza123 Could you simplify this

File: src/app/cdk/modal/modal-header/modal-header.component.scss [hunk 3]
```diff
@@ -9,6 +9,8 @@
   align-items: center;
   width: 100%;
   height: 56px;
+  padding-left: 16px;
```
Comment (DanielPalafox) [inline src/app/cdk/modal/modal-header/modal-header.component.scss:12]: @leomendoza123 and also this

File: src/app/cdk/side-bar/modals/modal-email/modal-email.component.ts [hunk 4]
```diff
@@ -19,4 +56,44 @@ export class ModalEmailComponent implements OnInit {
...
+      })
+      group[email.value].valueChanges.subscribe((value) => {
+        console.log(value)
+      })
+    })
+    this.emailsForm = new FormGroup(group)
+  }
+
+  formToBackendJson(fromGroup: FormGroup) {
+    // this._recordEmails.postEmails(this.backendJson)
```
Comment (DanielPalafox) [inline src/app/cdk/side-bar/modals/modal-email/modal-email.component.ts:97]: @leomendoza123 Is this supposed to be commented?

File: src/app/cdk/side-bar/modals/modal-email/modal-email.component.html [hunk 5]
```diff
@@ -10,10 +10,91 @@
...
-  <p>ORCID works best when you have multiple email addresses.</p>
-  <p>
-    Per the ORCID terms of use you may only add email addresses that you have
-    control over. Each time you add an email address we will send you an email
-    asking you to verify it.
-  </p>
+  <div class="container" [formGroup]="emailsForm">
+    <div class="row">
+      <div class="col">
+        <p>ORCID works best when you have multiple email addresses.</p>
```
Comment (DanielPalafox) [inline src/app/cdk/side-bar/modals/modal-email/modal-email.component.html:16]: @leomendoza123 All attributes i18n are missing in this file

File: src/app/cdk/side-bar/modals/modal-country/modal-country.component.ts [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+            displayIndex: i + 1,
+            visibility: {
+              visibility,
+            },
+          } as Address)
+        }
+      })
+    return countries
+
+    //  const i = this.countries.findIndex((value) => value.putCode === putcode)
```
Comment (DanielPalafox) [inline src/app/cdk/side-bar/modals/modal-country/modal-country.component.ts:133]: @leomendoza123 I don't think this is supposed to be here

**Review by DanielPalafox — CHANGES_REQUESTED** [review]
@ leomendoza123 Great job Leo !!. I left you some comments.

File: src/app/cdk/side-bar/modals/modal-email/modal-email.component.html (hunk 5 above)
Comment (leomendoza123) [reply src/app/cdk/side-bar/modals/modal-email/modal-email.component.html:16]: Thanks for noticing Dani. I will solve this as part of https://trello.com/c/4EOdjRy6/7214-speedup-bio-emails-5

**Review by DanielPalafox — APPROVED** [review]

**Pull request merged**